src/signal.rs

```diff
@@ -129,32 +99,27 @@ fn SIGRTMAX() -> c_int {
 
 /// Verify that a signal number is valid.
 ///
-/// VCPU signals must be enclosed within the OS
-/// limits for realtime signals. The remaining ones need
+/// Usually, VCPU signals are assigned and must be enclosed within the OS
+/// limits for realtime signals, while the remaining ones need
 /// to be between the minimum (`SIGHUP`) and maximum (`SIGSYS`) values.
 ///
 /// # Arguments
 ///
 /// * `num`: the signal number to be verify.
-/// * `for_vcpu`: if `num` is a vcpu signal.
 ///
 /// # Examples
 ///
 /// ```
 /// extern crate vmm_sys_util;
 /// use vmm_sys_util::signal::validate_signal_num;
 ///
-/// let num = validate_signal_num(1, true).unwrap();
+/// let num = validate_signal_num(1).unwrap();
 /// ```
-pub fn validate_signal_num(num: c_int, for_vcpu: bool) -> errno::Result<c_int> {
-    if for_vcpu {
-        let actual_num = num + SIGRTMIN();
-        if actual_num <= SIGRTMAX() {
-            return Ok(actual_num);
-        }
-    } else if SIGHUP <= num && num <= SIGSYS {
+pub fn validate_signal_num(num: c_int) -> errno::Result<c_int> {
+    if SIGHUP <= num && num <= SIGRTMAX() {
```

Comment:
The `[SIGRTMIN, SIGRTMAX]` range varies depending on arch and libc implementation. There might be a gap between the 32 standard signal numbers and the 32 real-time signal numbers.

```suggestion
    if (SIGHUP <= num && num <= SIGUSR2) || (SIGRTMIN() <= num && num <= SIGRTMAX()) {
```